app/utils/constants.ts

```diff
@@ -0,0 +1,105 @@
+import { Ruleset } from "../interfaces/Ruleset";
+
+export const TIME_OF_DAY_REGEX = new RegExp(/^([1-9]|1[0-2]):[0-5][0-9] (AM|PM)$/);
+export const DAYS_OF_WEEK = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"];
+export const TIME_OF_DAY_INCREMENTS = [
```

Comment:
Ah very nice :) was annoying having to scroll past all these each time haha, like this new layout

Reply:
Oh for sure keeps it more central and easier for re-use!